test/compound/live/TestEthLive.t.sol

```diff
@@ -1,106 +0,0 @@
-// SPDX-License-Identifier: GNU AGPLv3
-pragma solidity ^0.8.0;
-
-import "../setup/TestSetupVaultsLive.sol";
-
-contract TestEthLive is TestSetupVaultsLive {
```

Comment:
Why do we remove this one ?

Reply:
Because the are only testing the harvest vaults which are abandoned

Reply:
Ok